working-with-react.md

```diff
@@ -53,7 +46,7 @@ If you have a view that's orchestrating a lot of components, keep the state in t
 This means DON'T try to refactor an entire view hierarchy in one go, pick a small section of it and convert it to a react component, deploy that and then move on from there. More on this in this [excellent talk by Ryan Florence](https://www.youtube.com/watch?v=BF58ZJ1ZQxY).
 
 ### DO separate your markup from your logic within your components:
-It should be easy to see from your render method the composition of components and markup that will be rendered to the page.
+It should be easy to see from what your component renders, the composition of components and markup that will be rendered to the page.
```

Comment:
Slight rewording here because in the newer style of functional React components, there is no explicit `render()` method. Instead we just return JSX.